gui.py

```diff
@@ -78,30 +78,42 @@
     ],
 )
 
-# Control which allows user to specify a date range.
-# Mark the steps by day of year.
-date_ranges = [91, 121, 152, 182, 213, 243, 275]
-date_names = list(
-    map(lambda x: datetime.strptime(str(x), "%j").strftime("%-B"), date_ranges)
-)
-date_marks = dict(zip(date_ranges, date_names))
-range_slider = dcc.RangeSlider(
-    id="day_range",
-    marks=date_marks,
-    count=1,
-    min=91,
-    max=275,
-    step=1,
-    pushable=45,  # minimum date range span
-    value=[luts.default_date_range[0], luts.default_date_range[1]],
-)
-range_slider_field = html.Div(
-    className="field",
-    children=[
-        html.Label("Select date range", className="label"),
-        html.Div(className="control", children=[range_slider]),
-    ],
-)
+
+def get_day_range_slider(element_id):
+    """
+    Helper to create day range sliders.
+    `id` becomes the ID of the slider element.
+    """
+
+    # Control which allows user to specify a date range.
+    # Mark the steps by day of year.
+    date_ranges = [91, 121, 152, 182, 213, 243, 275]
```

Comment:
Think changing 243 to 244 on this line should make that date mark September 1.

Reply:
Fixed by improving the DOY handling in general!